### PR TITLE
Created Expanding Cards

### PR DESCRIPTION
**Pull request by gaurav24072002**
Hey !!
Please assign me this project under GSSOC '21

### CODE REVIEW

File: public/Expanding Cards [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EXPANDING CARDS</title>
+    <!-- <link rel="stylesheet" href="style.css"> -->
```
Comment (gaurav24072002) [inline public/Expanding Cards:8]: Remove this comment as I have attached the style tag separately.

File: public/Expanding Cards [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    })
+})
+
+function removeActiveClass(){
+    panels.forEach((panel) => {
+        panel.classList.remove('active');     
+    });
+}
+    </script>
+    <!-- <script src="script.js"></script> -->
```
Comment (gaurav24072002) [inline public/Expanding Cards:101]: Remove this also as script tag attach separately

**Comment by Vishal-raj-1** [comment]
@gaurav24072002 welcome to this project. first of all follow PR template, describe your changes, show your changes your github pages. second thing follow structure of the project, make seperate css,js file.

**Comment by Vishal-raj-1** [comment]
@gaurav24072002 ??

**Comment by gaurav24072002** [comment]
Yes , Sorry for delays I am on it and will solve all the issues by tommorow

**Comment by Vishal-raj-1** [comment]
@gaurav24072002 ?

**Comment by gaurav24072002** [comment]
Could you please merge my PR of #848  as my GitHub is showing some problem so that I can again clone the repository.

**Pull request closed without merging**